src/molecules/ChoosePayment.jsx

```diff
@@ -21,6 +21,10 @@ const SubHeaderSection = styled.div`
 `;
 
 export default function ChoosePayment() {
+  function handleClick(){
+    console.log("payment")
```

Comment:
El `console.log` nos servira en el dia a dia de desarrollo, pero no deberia estar en produccion, podemos tambien dejarle asociado in TODO, para darle funcionalidad a dicho handle.

```js
// TODO: ISSUE-01 Crear funcionalidad para elegir el método de pago al hacer click.
console.log("payment")
```